transactions/src/main/java/org/jboss/as/txn/service/TxnServices.java

```diff
@@ -65,6 +65,9 @@
 
     public static final ServiceName JBOSS_TXN_HTTP_REMOTE_TRANSACTION_SERVICE = JBOSS_TXN.append("service", "http-remote");
 
+    public static final ServiceName JBOSS_TXN_CONTEXT_XA_TERMINATOR = JBOSS_TXN.append("JBossContextXATerminator");
```

Comment:
Are there still services that depend on JBOSS_TXN_XA_TERMINATOR and would that be a problem?